docs/tutorials/contract-deployment.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deploying an EMP Contract
+sidebar_label: Expiring Multi-Party
+---
+
+This tutorial will show you how to deploy an Expiring Multi Party (EMP) contract using Etherscan or existing deployment scripts. By following these instructions, you will be able to launch an EMP on the Kovan testnet or Ethereum mainnet.
+
+This is the high-level EMP deployment process that you will need to follow.
+
+- Verify that your desired [price identifier](https://docs.umaproject.org/governance/adding-price-id#what-is-a-price-identifier) and collateral type is available on the network you wish to deploy your contract to.
+- Write your contract parameters.
+- Deploy your contract with Etherscan OR
+- Deploy your contract with a deployment script.
+
+:::danger
+
+Before deploying a mainnet contract, it is highly recommended that you have tested liquidation and dispute bots ready to deploy. Without a network of liquidation and dispute bots, a mainnet contract is vulnerable to attacks.
+
+:::
+
+## Is your desired price identifier and collateral type supported?
+Before deploying a contract, you should verify that your desired price identifier and collateral currency is already approved on the network you are trying to deploy to.
+
+View approved price identifiers and currencies here:
+- [Approved mainnet collateral currencies](https://docs.umaproject.org/oracle/mainnet-info#current-dvm-parameters)
+- [Approved mainnet price identifiers](https://docs.umaproject.org/oracle/mainnet-info#current-dvm-parameters)
+- [Approved Kovan collateral currencies](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Whitelisted%20Collateral%20Currencies)
+- [Approved Kovan price identifiers](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Pricefeed%20Identifiers)
+
+To add a new price identifier or collateral currency to mainnet, please propose one to UMA voters by following the [UMIP process](https://docs.umaproject.org/governance/umips).
+
+To add a new price identifier or collateral currency to Kovan, please follow these [instructions](https://docs.umaproject.org/governance/adding-price-id#adding-a-price-identifier-to-the-kovan-testnet) or contact the UMA core team in Discord.
+
+## EMP parameters
+
+The mandatory EMP parameters are:
+
+- `expirationTimestamp`: The UNIX timestamp that the EMP will expire at. Example December 31, 2024 expiry: `1735682400`.
```

Comment:
perhaps link to https://www.unixtimestamp.com/ to help with finding the associated timestamp for a given date.

Reply:
Good idea!